### DISCUSSION TITLE
When shall we start the Meetups?

### DISCUSSION THREAD

**Discussion by szabgab** [szabgab/rust.org.il#1] [Polls]
Traditionally most meetups are in the afternoon/evening hours between 18:00-20:00, though during Covid I ran run Zoom-based evens on Friday morning as well. (For Israelis that's the weekend). Someone recently suggested to start the meetings earlier or later. So I try to run a poll and as an experiment I'll use this GitHub Discussion for that.
I have option both for Zoom-based and in-person meetings. The latter most likely, when the security situation allows, will happen in Gush Dan.

Please upvote the answer(s) that are preferable for you and add more comments if you have other ideas.

I wanted the use the poll feature of GitHub, but only after people started to vote I realized you can only select one option here. So I removed the options and added comments with the options. Please :+1: on them! Select as many as you like. Sorry for the mess I created. 

In any case please check out our web site at https://rust.org.il/ where you can find a number of communication channels (eg. mailing list, Discord, Telegram) You are welcome to register on those.

**Comment by jondot** [comment]
If possible to also have the option for `Zoom weekday 9am-12pm` (anywhere in the range 9am to 12pm)
+1 for that from me.

**Comment by shaib** [comment]
My first preference is for in-person, 18:00, but if we're doing a vidconf, I'd prefer it later (I'd want to have separate selections)

**Reply by szabgab** [reply]
I just realized this is a radiobutton and you can select only one. That wasn't my intention

**Comment by szabgab** [comment]
Zoom weekday 17:00-19:00

**Comment by szabgab** [comment]
Zoom weekday 18:00-20:00

**Comment by szabgab** [comment]
Zoom weekday 19:00-21:00

**Comment by szabgab** [comment]
Zoom weekday 20:00-22:00

**Comment by szabgab** [comment]
In person weekday 17:00-19:00

**Comment by szabgab** [comment]
In person weekday 18:00-20:00

**Comment by szabgab** [comment]
In person weekday 19:00-21:00

**Comment by szabgab** [comment]
In person weekday 20:00-22:00

**Comment by szabgab** [comment]
Zoom Friday morning hours

**Comment by IdanLib** [comment]
So, as someone with a full-time and a small child, the logic is this:
If it's zoom, better later, 19:00 at earliest (we have enough time to have dinner with the kid, bath and storytime). 17:00 might also be ok if session is 1 hours. 

If in person, the opposite is true - start at 17:00 at latest, so we could come directly from work, and leave at 18:45-19:00 to get home in time for storytime. 

I personally like Fri mornings free but I can do them every once in a while. If they start early (at 9:00. say), that's better.
